tests/testthat/test-as-data-frame.R

```diff
@@ -96,31 +110,12 @@ with_mock_crunch({
         expect_equal(names(new_ds_df),
                      aliases(variables(new_ds)))
         expect_equal(ncol(new_ds_df), 6)
-        expect_silent(
-            expect_equal(names(as.data.frame(new_ds_df)),
-                         c("birthyr", "location", "subvar2", "subvar1", 
-                           "subvar3", "textVar", "starttime", "subvar2", 
-                           "subvar1", "subvar3")))
-        
+
         # now we want the hidden vars to be includes
         new_ds_df <- as.data.frame(new_ds, include.hidden = TRUE)
         expect_equal(names(new_ds_df),
                      aliases(allVariables(new_ds)))
         expect_equal(ncol(new_ds_df), 7)
-        expect_warning(
-            expect_equal(names(as.data.frame(new_ds_df)),
-                         c("birthyr", "gender", "location", "subvar2", 
-                           "subvar1", "subvar3", "textVar", "starttime",
-                           "subvar2", "subvar1", "subvar3")),
-            "Variable gender is hidden")
-    })
-    
-    test_that("as.data.frame size limit", {
-        with(temp.option(crunch.data.frame.limit=50), {
-            expect_error(as.data.frame(ds, force=TRUE),
-                "Dataset too large to coerce")
-            expect_true(is.data.frame(as.data.frame(ds[,1:2], force=TRUE)))
-        })
```

Comment:
Same as the comment above: it would be good to be able to keep these tests running even without an API.